<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Daniel Chang</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <!-- <link rel="icon" type="image/png" href="images/favicon.png"> -->

</head>
<body style="background-color:#f9f7ee">

  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">
    <div class="row", style="margin-top: 5%">

      <!-- Header -->
      <div class="row">
        <!-- Photo -->
        <div class="three columns", style="margin-left: 9%">
            <h4>Daniel Chang</h4>
            <img src="images/profile.jpeg", style="height: 180px", class="ImageBorder"></img>
        </div>

        <!-- Hello -->
        <div class="five columns", style="margin-top: 6%; margin-left: 2%">
          <h6>
            Hello! I'm currently a master's computer science student at the University of Minnesota. I research computational biology.
          </h6>
        </div>

        <!-- Social media & contact -->
        <div class="five columns", style="margin-top: 0.5%">
          <!-- Column 1 -->
          <div class="eight columns">
            <p>
              <img src="images/email.svg", style="height: 16px"></img> chan1975 [at] umn.edu <br>
              <!-- <a href="mailto:maxwshen@gmail.com"> <img src="images/email.png", style="height: 16px"></img> maxwshen@gmail.com</a> <br> -->
              <a href="https://scholar.google.com/citations?user=nwQ70xkAAAAJ&hl=en"> <img src="images/google-scholar.svg", style="height: 16px"></img> Google Scholar</a> <br>
              <img src="images/location.png", style="height: 16px"></img> Minneapolis, MN, USA
            </p>
          </div>
          <!-- Column 2 -->
          <div class="four columns">
            <p>
              <a href="files/daniel_chang_cv.pdf"> <img src="images/document.png", style="height: 16px"></img> CV </a> <br>
              <a href="https://github.com/danielchang2002"> <img src="images/GitHub-Mark-120px-plus.png", style="height: 16px"></img> GitHub</a> <br>
              <!-- <a href="https://www.linkedin.com/in/maxwshen/"> <img src="images/LI-In-Bug.png", style="height: 16px"></img> LinkedIn</a> <br> -->
              <a href="https://www.linkedin.com/in/daniel-chang-b93473204/"> <img src="images/LI-In-Bug.png", style="height: 16px"></img> LinkedIn</a> <br>
            </p>
          </div>
        </div>
      </div>

      <div class="nine columns">

        <!-- <div class="four columns"> -->

        <!-- </div> -->

        <!-- <div class="eight columns"> -->
        <h6>
          <!-- As a computationalist at heart, I believe in interdisciplinarity, getting one's feet wet, and that better solutions to real-world problems arise by marrying each problem's unique structure with thoughtful modeling and inference design. -->

        </h6>
        <!-- </div> -->

        <h6>
          <!-- How can we get the best of deep learning and human scientific knowledge? How can we build models with improved generalization, and can build models that generalize in systematic ways that we can understand? I am interested in exploring these questions with representation learning of causal structure, expanding the expressivity of deep models with tools like the implicit function theorem, controlled generative modeling, curvature-aware optimization, and invariant causal prediction and invariant risk minimization. -->
          <!-- Science is an interesting domain for ML we often have more truth than quality data, unlike ML for general intelligence domains like vision and language. -->
        </h6>

        <h6>
            I'm currently working with <a href="http://csbio.cs.umn.edu/">Chad Myers</a> on developing machine learning models for cancer genetic dependency prediction.
            I received a B.S., <i>summa cum laude</i>, from the University of Minnesota in computer science in 2023.
            While there, I worked with <a href="https://www.sung-lab.org/">Jaeyun Sung</a> on translating gut microbiome data into health insights, and <a href="https://www.knightslab.org/">Dan Knights</a> on dissecting the structure and function of the infant gut microbiome.
        </h6>

        <!-- <h6>
          Service: Area chair and co-organizer of ICLR Workshop on Machine Learning for Drug Discovery, 2022 & 2023.
        </h6> -->

        <!-- Social media -->
        <div class="row">


        </div>

        <!-- Publications -->
        <div>
          <hr>

          <h5>Publications</h5>

          <p>
            * - co-first author; # - co-corresponding author
          </p>

          <p><b>Submitted, under review, or in revision</b></p>

          <p>
            <b>Daniel Chang</b>, Xiang Zhang, and Chad Myers. "Ridge Regression Baseline Model Outperforms Deep Learning Method for Cancer Genetic Dependency Prediction". submitted. 
            [<a href="https://doi.org/10.1101/2023.11.29.569083">bioRxiv</a>][<a href="https://github.com/danielchang2002/deepdep_reanalysis">code</a>]
          </p>

          <p>
            <img src="images/poop.png", style="height: 200px", class="ImageBorder"></img> <img src="images/gmwi2_fig.png", style="height: 200px", class="ImageBorder"></img>  <br>
            <b>Daniel Chang</b>*, Vinod K. Gupta*, Benjamin Hur, Sergio Cobo-Lopez, Kevin Y. Cunningham, Levi M. Teigen, Erin L. Longbrake, Insuk Lee, Yong-Su Jin, Nam Soo Han, John M. Davis III, and Jaeyun Sung. 
            Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles". 
            in revision at <i>Nature Communications</i>. 
            [<a href="https://doi.org/10.1101/2023.09.30.560294">bioRxiv</a>][<a href="https://github.com/danielchang2002/GMWI2">code</a>]
          </p>

          <p>
            <b>Published or in press</b>
          </p>

          <p>
            Dong Hyeon Lee*, Hyunbin Seong*, <b>Daniel Chang</b>, Vinod K. Gupta, Jiseung Kim, Seongwon Cheon, Geonhee Kim, Jaeyun Sung#, and Nam Soo Han#. 
            "Evaluating the Prebiotic Effect of Oligosaccharides on Gut Microbiome Wellness Using in vitro Fecal Fermentation". <i>npj Science of Food</i> (2023).
            [<a href="https://doi.org/10.1038/s41538-023-00195-1">html</a>][<a href="files/prebiotics.pdf">pdf</a>]
          </p>

          <p>
            <img src="images/gmwi-webtool_fig1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/gmwi-webtool_fig2.png", style="height: 200px", class="ImageBorder"></img>  <br>
            <b>Daniel Chang</b>, Vinod K. Gupta, Benjamin Hur, Kevin Y. Cunningham, and Jaeyun Sung. 
            "GMWI-webtool: A User-friendly Browser Application for Assessing Health through Metagenomic Gut Microbiome Profiling". <i>Bioinformatics</i> (2023). 
            [<a href="https://doi.org/10.1093/bioinformatics/btad061">html</a>][<a href="files/gmwi-webtool.pdf">pdf</a>][<a href="https://gmwi-webtool.github.io/">code</a>]
          </p>

          <p>
            Vinod K. Gupta, Utpal Bakshi, <b>Daniel Chang</b>, Aileen R. Lee, John M. Davis III, Sriram Chandrasekaran, Yong-Su Jin, Michael F. Freeman, and Jaeyun Sung. 
            "TaxiBGC: a Taxonomy-guided Approach for Profiling Experimentally Characterized Microbial Biosynthetic Gene Clusters in Metagenomes". <i>mSystems</i> (2022). 
            [<a href="https://doi.org/10.1128/msystems.00925-22">html</a>][<a href="files/taxibgc.pdf">pdf</a>][<a href="https://github.com/danielchang2002/TaxiBGC_2022/tree/main">code</a>]
          </p>

        <!-- Talks -->
        <!-- <div>
          <hr>
          <h5>Recognition</h5>
          <p>
            Invited talk at Models, Inference and Algorithms at the Broad Institute. Spring 2020 — delayed. <br>
            Invited talk at Synthego, Jan. 2019. <br>
            Selected talk (top 15%) at Biology of Genomes, Cold Spring Harbor, May 2018. <br>
            NSF Graduate Research Fellowship (top 15%), 2015. <br>
          </p>
        </div> -->
        <div>
          <hr>
          <h5>Presentations</h5>
          <p>
            <img src="images/magic_beta.png", style="height: 200px", class="ImageBorder"></img> <img src="images/magic_network.png", style="height: 200px", class="ImageBorder"></img>  <br>
            <b>Daniel Chang</b>, Suzie Hoops, Dan Knights. "Metagenomic MAGIC: Dissecting the Structure and Function of the Developing Gut Microbiome". 
            <i>Minnesota Supercomputing Institute Research Exhibition</i>. Minneapolis, MN. Apr 25, 2023. (Poster Award - $1,000) [<a href="https://drive.google.com/file/d/1gkQai9Bw95D7pwG8oa6yd5MvfxK9jzlB/view?usp=drive_link">pdf</a>]
          </p>

          <p>
            <b>Daniel Chang</b>, Vinod K. Gupta, Jaeyun Sung. "Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles" <i>Cold Spring Harbor Microbiome Meeting</i>. 
            Laurel Hollow, NY. Oct 27, 2022.
          </p>
        </div>

        <!-- Teaching -->
        <div>
          <hr>
          <h5>Teaching</h5>
          <p>
            Teaching Assistant, CSCI 5481: Computational Techniques for Genomics, Prof. Dan Knights
            <br>
            University of Minnesota, Spring 2023
          </p>
        </div>

        <!-- Skills -->
        <!-- <div>
          <hr>
          <h5>Skills</h5>
          <p>
            <u>Applied Machine Learning</u>: My work customizes powerful, modern deep models to leverage the unique structure within each real world problem. I have designed <a href="https://github.com/maxwshen/be_predict_bystander">deep conditional autoregressive models to model base editing outcomes</a>, and <a href="https://github.com/maxwshen/inDelphi-model">jointly-trained multitask sister deep networks to accurately learn a particularly noisy subset of CRISPR editing activity</a>. I have taught classes to MIT undergrad and graduate students on <a href="https://github.com/maxwshen/iap-appbml">Bayesian modeling</a>, <a href="https://github.com/maxwshen/iap-cidl">deep learning, and causal inference</a>. I am proficient in pytorch, and enjoy keeping up with modern toolkits: see <a href="https://github.com/maxwshen/latent-multiGP-inference">my low-level integration between pyro and gpytorch to learn 10 latent Gaussian processes on time series data in a larger probabilistic model with stochastic variational inference</a>.
          </p>

          <p>
            <u>Software Engineering</u>: See my <a href="https://github.com/maxwshen">GitHub</a>. I completed software engineering internships at Qualcomm Korea (2013) and Illumina (2014). Python is my language of choice, though I have previously worked in C++. Summa cum laude B.S. in Computer Science with a specialization in bioinformatics (2011-2015). 
          </p>

          <p>
            <u>Data Visualization</u>: My interactive <a href="https://www.crisprindelphi.design">web</a> <a href="https://www.crisprbehive.design">apps</a> have received <a href="https://plotly.com/resources/indelphi/">press attention from Dash plotly</a>. I am proficient in Adobe Illustrator, Photoshop, Premiere Pro, and After Effects, and use matplotlib, pandas, and seaborn everyday. I am also proficient in html, css, and dash plotly.
          </p>

          <p>
            <u>Communication and Collaboration</u>: My Ph.D. has featured extensive collaboration with wet lab experimentalists, including <a href="http://sherwoodlab.bwh.harvard.edu/">Richard I. Sherwood</a> and <a href="https://www.researchgate.net/profile/Mandana_Arbab">Mandana Arbab</a>. I have completed a 40 h course on conflict management and mediation that has substantially impacted my life.
          </p>

          <p>
            <u>Management</u>: For two years in undergrad, I managed 9 teams with a total of ~100 students to host a regional urban-style (hip hop) dance competition with ~2,000 audience members with revenue and expenses of $35,000/year. Each team had ~10 students and 2 team leaders, and all team leaders were overseen by me and one other co-leader. This provided substantial public speaking experience, as I led dozens of meetings speaking to and motivating our team of 100 students.
          </p>

        </div> -->

        <!-- Footer -->
        <div class="row" style="text-align:center;">
          <hr>
          <p>
            <a href="https://github.com/danielchang2002/danielchang2002.github.io"> <img src="images/GitHub-Mark-120px-plus.png", style="height: 16px"></img> source</a> <br>
          </p>
          <!-- <div class="seven columns">
            <p>
              Max W. Shen is a Principal Scientist at Genentech.
            </p>
            <p style="color: gray">
              This website was designed with <a href="http://getskeleton.com/">Skeleton CSS</a>.

            </p>
          </div> -->

          <!-- <div class="three columns">
            <p>
              <a href="mailto:maxwshen@gmail.com">maxwshen@gmail.com</a> <br>
              <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ">Google Scholar</a> <br>
              <a href="https://www.linkedin.com/in/maxwshen/">LinkedIn</a> <br> 
              <a href="https://github.com/maxwshen">GitHub</a> <br>
              <a href="https://twitter.com/maxwshen">Twitter</a>  <br>
            </p>
          </div> -->

        </div>

      </div>
    </div>
  </div>

<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
</body>
</html>