<!DOCTYPE html>
<html lang="en">
<head>

  <!-- Basic Page Needs
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta charset="utf-8">
  <title>Daniel Chang</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <!-- Mobile Specific Metas
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- FONT
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">

  <!-- CSS
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">

  <!-- Favicon
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <!-- <link rel="icon" type="image/png" href="images/favicon.png"> -->

</head>
<body style="background-color:#f9f7ee">

  <!-- Primary Page Layout
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
  <div class="container">

      <!-- Header -->
      <div class="row", style="display: flex; margin-top: 5%; justify-content: center">

        <!-- Photo -->
        <div class="four columns">
            <img src="images/headshot_2023_square.jpg", style="height: 210px", class="ImageBorder"></img>
        </div>

        <!-- Hello -->
        <div class="six columns" style="margin-left:0">

          <h4>Daniel Chang</h4>

          <h6>
            Hello! I'm a PhD student at <a href="https://med.stanford.edu/genetics.html">Stanford Genetics</a>. I research computational biology.
          </h6>
                  <a href="mailto:changdan@stanford.edu"><img src="images/cemail.svg", style="height: 40px"></img></a>
                  <a href="https://scholar.google.com/citations?user=nwQ70xkAAAAJ&hl=en"> <img src="images/scholar.svg", style="height: 40px"></img></a>
                  <a href="files/cv_5_14_2025.pdf"> <img src="images/document.svg", style="height: 40px"></img></a>
                  <a href="https://github.com/danielchang2002"> <img src="images/cgithub.svg", style="height: 40px"></img></a>
                  <a href="https://www.linkedin.com/in/daniel-chang-b93473204/"> <img src="images/LinkedIn_icon_circle.svg", style="height: 40px"></img></a>
                  <a href="https://www.youtube.com/@daniel_chang2002"> <img src="images/youtube.svg", style="height: 40px"></img></a>
        </div>
      </div>


      <br>
      <div class="row" style="margin-top: 1%">
        <div class="twelve columns">
          <h6>
              I'm currently doing PhD rotations!
              I graduated <i>summa cum laude</i> with a BS/MS in computer science from the University of Minnesota in 2024.
              While there, I worked with <a href="http://csbio.cs.umn.edu/">Chad Myers</a> on developing machine learning models for cancer genetic dependency prediction,
              <a href="https://www.sung-lab.org/">Jaeyun Sung</a> on translating gut microbiome data into health insights, 
              and <a href="https://www.knightslab.org/">Dan Knights</a> on dissecting the structure and function of the infant gut microbiome.
          </h6>

          <!-- Publications -->
          <div>
            <hr>

            <h5>Publications</h5>

            <p>
              */** - equal contribution; # - co-corresponding author
            </p>

            <!-- <p><b>Submitted, under review, or in revision</b></p> -->

            <!-- <p>
              <b>Published or in press</b>
            </p> -->

            <p>
                <img src="images/phage_fig1.png", style="height: 200px", class="ImageBorder"></img> 
                <img src="images/IScream.png", style="height: 200px", class="ImageBorder"></img> 
                <img src="images/IS30_phylogeny.png", style="height: 200px", class="ImageBorder"></img> 
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2025.05.09.652943v1">Discovering Broader Host Ranges and an IS-bound Prophage Class Through Long-Read Metagenomics.</a>
              Jakob Wirbel*, Angela S. Hickey*, <b>Daniel Chang</b>, Nora J. Enright, Mai Dvorak, Rachael B. Chanin, Danica T. Schmidtke, and Ami S. Bhatt.
              <i>bioRxiv</i> (2025). 
              [<a href="https://x.com/danielchang2002/status/1922096900745920724">thread</a>]
            </p>

            <p>
                <img src="images/evo2_fig1cd.png", style="height: 200px", class="ImageBorder"></img> 
                <!-- <img src="images/evo2_fig1b.png", style="height: 200px", class="ImageBorder"></img>  -->
                <!-- <img src="images/gmwi2_fig1.png", style="height: 200px", class="ImageBorder"></img>  -->
                <img src="images/evo2_mech_interp.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              <a href="https://www.biorxiv.org/content/10.1101/2025.02.18.638918v1">Genome modeling and design across all domains of life with Evo 2.</a>
              Garyk Brixi*, Matthew Durrant*, Jerome Ku*, Michael Poli*, Greg Brockman**, <b>Daniel Chang**</b>, Gabriel Gonzalez**, Samuel King**, David Li**, Aditi Merchant**, Mohsen Naghipourfar**, Eric Nguyen**, Chiara Ricci-Tam**, David Romero**, Gwanggyu Sun**, Ali Taghibakshi**, Anton Vorontsov**, Brandon Yang**, et al., Dave Burke, Hani Goodarzi, Patrick Hsu#, and Brian Hie#.
              <i>bioRxiv</i> (2025). 
              [<a href="https://github.com/ArcInstitute/evo2">code</a>][<a href="https://arcinstitute.org/news/blog/evo2">blog</a>][<a href="https://x.com/danielchang2002/status/1892722520618316180">thread</a>]
            </p>

            <p>
                <img src="images/poop.png", style="height: 200px", class="ImageBorder"></img> 
                <img src="images/gmwi2_fig1.png", style="height: 200px", class="ImageBorder"></img> 
                <img src="images/gmwi2_fig5.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              <a href="https://www.nature.com/articles/s41467-024-51651-9">Gut Microbiome Wellness Index 2 Enhances Health Status Prediction from Gut Microbiome Taxonomic Profiles.</a>
              <b>Daniel Chang</b>*, Vinod K. Gupta*, Benjamin Hur, Sergio Cobo-López, Kevin Y. Cunningham, Nam Soo Han, Insuk Lee, Vanessa L. Kronzer, Levi M. Teigen, Lioudmila V. Karnatovskaia, Erin E. Longbrake, John M. Davis III, Heidi Nelson, and Jaeyun Sung.
              <i>Nature Communications</i> (2024). 
              [<a href="files/gmwi2.pdf">pdf</a>][<a href="https://github.com/danielchang2002/GMWI2">code</a>][<a href="https://newsnetwork.mayoclinic.org/discussion/mayo-researchers-develop-tool-that-measures-health-of-a-persons-gut-microbiome/?sf199954604=1">blog</a>]
            </p>

            <p>
                <img src="images/deepdep_fig1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/deepdep_fig2.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              <a href="https://doi.org/10.1101/2023.11.29.569083">Ridge Regression Baseline Model Outperforms Deep Learning Method for Cancer Genetic Dependency Prediction.</a>
              <b>Daniel Chang</b>, Xiang Zhang, and Chad Myers. 
              <i>bioRxiv</i> (2023). 
              [<a href="https://github.com/danielchang2002/deepdep_reanalysis">code</a>]
            </p>


            <p>
                <img src="images/prebiotics_fig1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/prebiotics_fig3.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              <a href="https://doi.org/10.1038/s41538-023-00195-1">Evaluating the Prebiotic Effect of Oligosaccharides on Gut Microbiome Wellness Using in vitro Fecal Fermentation.</a>
              Dong Hyeon Lee*, Hyunbin Seong*, <b>Daniel Chang</b>, Vinod K. Gupta, Jiseung Kim, Seongwon Cheon, Geonhee Kim, Jaeyun Sung#, and Nam Soo Han#. 
              <i>npj Science of Food</i> (2023).
              [<a href="files/prebiotics.pdf">pdf</a>]
            </p>

            <p>
              
                <img src="images/gmwi-webtool_fig0.png", style="height: 200px", class="ImageBorder"></img> <img src="images/gmwi-webtool_fig1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/gmwi-webtool_fig2.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              <a href="https://doi.org/10.1093/bioinformatics/btad061">GMWI-webtool: A User-friendly Browser Application for Assessing Health through Metagenomic Gut Microbiome Profiling.</a>
              <b>Daniel Chang</b>, Vinod K. Gupta, Benjamin Hur, Kevin Y. Cunningham, and Jaeyun Sung. 
              <i>Bioinformatics</i> (2023). 
              [<a href="files/gmwi-webtool.pdf">pdf</a>][<a href="https://gmwi-webtool.github.io/">code</a>]
            </p>

            <p>
              <img src="images/taxibgc_fig1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/taxibgc_fig5.png", style="height: 200px", class="ImageBorder"></img> <img src="images/taxibgc_fig6.png", style="height: 200px", class="ImageBorder"></img>  <br>
              <a href="https://doi.org/10.1128/msystems.00925-22">TaxiBGC: a Taxonomy-guided Approach for Profiling Experimentally Characterized Microbial Biosynthetic Gene Clusters in Metagenomes.</a>
              Vinod K. Gupta, Utpal Bakshi, <b>Daniel Chang</b>, Aileen R. Lee, John M. Davis III, Sriram Chandrasekaran, Yong-Su Jin, Michael F. Freeman, and Jaeyun Sung. 
              <i>mSystems</i> (2022). 
              [<a href="files/taxibgc.pdf">pdf</a>][<a href="https://github.com/danielchang2002/TaxiBGC_2022/tree/main">code</a>]
            </p>

          <!-- Talks -->
          <!-- <div>
            <hr>
            <h5>Recognition</h5>
            <p>
              Invited talk at Models, Inference and Algorithms at the Broad Institute. Spring 2020 — delayed. <br>
              Invited talk at Synthego, Jan. 2019. <br>
              Selected talk (top 15%) at Biology of Genomes, Cold Spring Harbor, May 2018. <br>
              NSF Graduate Research Fellowship (top 15%), 2015. <br>
            </p>
          </div> -->
          <div>
            <hr>
            <h5>Presentations</h5>
            <p>
              <img src="images/magic_beta1.png", style="height: 200px", class="ImageBorder"></img> <img src="images/magic_network0.png", style="height: 200px", class="ImageBorder"></img> <img src="images/magic_network.png", style="height: 200px", class="ImageBorder"></img> 
              <br>
              <a href="https://drive.google.com/file/d/1gkQai9Bw95D7pwG8oa6yd5MvfxK9jzlB/view?usp=drive_link">Metagenomic MAGIC: Dissecting the Structure and Function of the Developing Gut Microbiome.</a>
              <b>Daniel Chang</b>, Suzie Hoops, and Dan Knights.
              <i>Minnesota Supercomputing Institute Research Exhibition</i>. Minneapolis, MN. Apr 25, 2023. (Poster Award - $1,000)
            </p>

            <p>
              Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles.
              <b>Daniel Chang</b>, Vinod K. Gupta, and Jaeyun Sung. 
              <i>Mayo Clinic Individualizing Medicine Conference.</i>
              Rochester, MN. Nov 2, 2022.
            </p>

            <p>
              Gut Microbiome Wellness Index 2 for Enhanced Health Status Prediction from Gut Microbiome Taxonomic Profiles.
              <b>Daniel Chang</b>, Vinod K. Gupta, and Jaeyun Sung. 
              <i>Cold Spring Harbor Microbiome Meeting</i>. 
              Laurel Hollow, NY. Oct 27, 2022.
            </p>

          </div>

          <!-- Teaching -->
          <div>
            <hr>
            <h5>Teaching</h5>
            <p>
              <img src="images/csci5481_hw1_sc.png", style="height: 200px", class="ImageBorder"></img> <img src="images/csci5481_hw2_sc.png", style="height: 200px", class="ImageBorder"></img>
              <br>
              Teaching Assistant, CSCI 5481: Computational Techniques for Genomics, Prof. Dan Knights, University of Minnesota, Spring 2023
            </p>
          </div>

          <!-- Skills -->
          <!-- <div>
            <hr>
            <h5>Skills</h5>
            <p>
              <u>Applied Machine Learning</u>: My work customizes powerful, modern deep models to leverage the unique structure within each real world problem. I have designed <a href="https://github.com/maxwshen/be_predict_bystander">deep conditional autoregressive models to model base editing outcomes</a>, and <a href="https://github.com/maxwshen/inDelphi-model">jointly-trained multitask sister deep networks to accurately learn a particularly noisy subset of CRISPR editing activity</a>. I have taught classes to MIT undergrad and graduate students on <a href="https://github.com/maxwshen/iap-appbml">Bayesian modeling</a>, <a href="https://github.com/maxwshen/iap-cidl">deep learning, and causal inference</a>. I am proficient in pytorch, and enjoy keeping up with modern toolkits: see <a href="https://github.com/maxwshen/latent-multiGP-inference">my low-level integration between pyro and gpytorch to learn 10 latent Gaussian processes on time series data in a larger probabilistic model with stochastic variational inference</a>.
            </p>

            <p>
              <u>Software Engineering</u>: See my <a href="https://github.com/maxwshen">GitHub</a>. I completed software engineering internships at Qualcomm Korea (2013) and Illumina (2014). Python is my language of choice, though I have previously worked in C++. Summa cum laude B.S. in Computer Science with a specialization in bioinformatics (2011-2015). 
            </p>

            <p>
              <u>Data Visualization</u>: My interactive <a href="https://www.crisprindelphi.design">web</a> <a href="https://www.crisprbehive.design">apps</a> have received <a href="https://plotly.com/resources/indelphi/">press attention from Dash plotly</a>. I am proficient in Adobe Illustrator, Photoshop, Premiere Pro, and After Effects, and use matplotlib, pandas, and seaborn everyday. I am also proficient in html, css, and dash plotly.
            </p>

            <p>
              <u>Communication and Collaboration</u>: My Ph.D. has featured extensive collaboration with wet lab experimentalists, including <a href="http://sherwoodlab.bwh.harvard.edu/">Richard I. Sherwood</a> and <a href="https://www.researchgate.net/profile/Mandana_Arbab">Mandana Arbab</a>. I have completed a 40 h course on conflict management and mediation that has substantially impacted my life.
            </p>

            <p>
              <u>Management</u>: For two years in undergrad, I managed 9 teams with a total of ~100 students to host a regional urban-style (hip hop) dance competition with ~2,000 audience members with revenue and expenses of $35,000/year. Each team had ~10 students and 2 team leaders, and all team leaders were overseen by me and one other co-leader. This provided substantial public speaking experience, as I led dozens of meetings speaking to and motivating our team of 100 students.
            </p>

          </div> -->

          <!-- Footer -->
          <div class="row" style="text-align:center;">
            <hr>
            <p>
              <a href="https://github.com/danielchang2002/danielchang2002.github.io"> <img src="images/cgithub.svg", style="height: 16px; vertical-align: middle;"></img> source</a> <br>
            </p>
            <!-- <div class="seven columns">
              <p>
                Max W. Shen is a Principal Scientist at Genentech.
              </p>
              <p style="color: gray">
                This website was designed with <a href="http://getskeleton.com/">Skeleton CSS</a>.

              </p>
            </div> -->

            <!-- <div class="three columns">
              <p>
                <a href="mailto:maxwshen@gmail.com">maxwshen@gmail.com</a> <br>
                <a href="https://scholar.google.com/citations?hl=en&user=9woFwYcAAAAJ">Google Scholar</a> <br>
                <a href="https://www.linkedin.com/in/maxwshen/">LinkedIn</a> <br> 
                <a href="https://github.com/maxwshen">GitHub</a> <br>
                <a href="https://twitter.com/maxwshen">Twitter</a>  <br>
              </p>
            </div> -->

          </div>

        </div>
    </div>
  </div>

<!-- End Document
  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
</body>
</html>